latex_access/tests/test_speech.py

```diff
@@ -0,0 +1,80 @@
+import unittest
+from parameterized import parameterized
+from latex_access import speech
+
+try:
+    test_speech = speech.speech()
+    test_speech.sqrt('{2^2}', 0)
+    incompatible = False
+except AttributeError:
+    incompatible = True
+
+class TestSpeech(unittest.TestCase):
+
+    def setUp(self):
+        """Creates a speech instance."""
+        self.speech = speech.speech()
+
+    def test_adding_file(self):
+        """Tests adding of speechtable file."""
+        self.assertEqual(self.speech.files, ['speech.table'])
+
+    def test_adding_to_table(self):
+        """Tests adding of new elements to speech table."""
+        self.assertEqual(self.speech.table['^'], self.speech.super)
+        self.assertEqual(self.speech.table['\\int'], self.speech.integral)
+        self.assertEqual(self.speech.table['\mathbb'], ("<bold>","</bold>"))
+
+    def test_space(self):
+        """Tests creation of space attribute."""
+        self.assertEqual(self.speech.space, ' ')
+
+    def test_super(self):
+        """Tests translation of superscripts."""
+        self.assertEqual(self.speech.super('2', 0), (' squared ', 1))
+        self.assertEqual(self.speech.super('3', 0), (' cubed ', 1))
+        self.assertEqual(self.speech.super('{\prime}', 0), (' prime ', 8))
+        self.assertEqual(self.speech.super('{10}', 0), (' to the <sup> 10 </sup> ', 4))
+
+    def test_sqrt(self):
+        """Tests translations of roots."""
+        self.assertEqual(self.speech.sqrt('2', 0), (' root 2', 1))
+        self.assertEqual(self.speech.sqrt('{16}', 0), (' root 16', 4))
+        self.assertEqual(self.speech.sqrt('[3]{27}', 0), ('cube root 27', 7))
+        self.assertEqual(self.speech.sqrt('[5]{32}', 0), ('5th root 32', 7))
+
+    @unittest.skipIf(incompatible, "This test fails in Python 3 because of StringType attribute not available in Types module.")
+    def test_complex_roots(self):
+        """Tests translation of complex roots. Separated from other tests because it fails on Python 3."""
+        self.assertEqual(self.speech.sqrt('{2^2}', 0), (' begin  root 2 squared  end root', 5))
+
+    def test_frac(self):
+        """Tests translactions of fractions."""
```

Comment:
nit: translactions -> translations
```suggestion
        """Tests translations of fractions."""
```